contracts/merklePath.scrypt

```diff
@@ -0,0 +1,42 @@
+
+struct Sibling {
```

Comment:
Sibling -> Node

Let's use node to be consistent w/ the community
https://github.com/bitcoin-sv-specs/merkle-proof-standard-example/blob/4f4b47149bc52c292328c3c9b114b8a919560d7a/verifyMerkleProofJSON.js#L63